external-crates/move/crates/move-compiler/tests/custom_rules/shift_overflow.move

```diff
@@ -0,0 +1,9 @@
+module 0x42::M {
+
+    fun func1(x: u64) {
+        let _b = x << 24;
+        let _b = x << 64; // <Issue:5>
+        let _b = x << 65; // <Issue:5>
+        let _b = x >> 66; // <Issue:5>
```

Comment:
I'd change this to just comment what the test is testing up above.

We should also have tests where the LHS is something other than `u64`, i.e. a test for each of the integer sizes

Reply:
Checkout new PR for this lint #16820